### PR TITLE
[Term Entry] JavaScript Dates .setUTCDate() method

### PR DESCRIPTION
**Pull request by nycolop**
### Description

- Added the .setUTCDate() method from JavaScript Date object.
- Corresponds issue: #2547 
- Added a new folder and file .md in this path: **docs/content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md**

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```pseudo
+myDate.setUTCDate(day)
+```
+
+The `.setUTCDate()` method receives a day argument representing a number between 1 to 31.
+
+## Example
+
+In this example, we create a Date object in a variable called `myDate`. We then use `.setUTCDate()` to modify the day to 10.
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md:27]:
```suggestion
In this example, a Date object is created in a variable called `myDate`. `.setUTCDate()` is then used to modify the day to 10.
```
try not to use pronouns

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
One change from my side

**Comment by SSwiniarski** [comment]
@nycolop tag me when you've addressed @dakshdeepHERE's comments and I'll do a first maintainer review.

File: content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md (hunk 1 above)
Comment (SSwiniarski) [reply content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md:27]: In particular we avoid using first- or second-person pronouns in Docs.

File: content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+Title: '.setUTCDate()'
+Description: 'Changes the day to a Date object in Coordinated Universal Time (UTC).'
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md:3]:
```suggestion
Description: 'Changes the day of a Date object in Coordinated Universal Time (UTC).'
```

File: content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Tags:
+  - 'Date'
+  - 'Methods'
+  - 'Time'
+CatalogContent:
+  - 'paths/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`.setUTCDate()`** method changes the day to a Date object.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md:15]:
```suggestion
The **`.setUTCDate()`** method changes the day of a Date object in Coordinated Universal Time (UTC)..
```

File: content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Syntax
+
+```pseudo
+myDate.setUTCDate(day)
+```
+
+The `.setUTCDate()` method receives a day argument representing a number between 1 to 31.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setUTCDate/setUTCDate.md:23]:
```suggestion
The `.setUTCDate()` method receives a `day` argument representing a number between 1 to 31.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@nycolop I committed @dakshdeepHERE's comment and did a first review to get this moving. Tag me when you've addressed my comments.

**Review by SSwiniarski — APPROVED** [review]

**Review by caupolicandiaz — APPROVED** [review]
@nycolop @SSwiniarski Made some minor edits. Looks good to go

**Pull request merged**